lib/dns/records.js

```diff
@@ -850,8 +908,8 @@ class SMIME extends Struct {
     assert((json.usage & 0xff) === json.usage);
     assert((json.selector & 0xff) === json.selector);
     assert((json.matchingType & 0xff) === json.matchingType);
-    assert(typeof json.fingerprint === 'string');
-    assert((json.fingerprint.length >>> 1) <= 255);
+    assert(typeof json.certificate === 'string');
```

Comment:
TODO: break this into its own PR

Reply:
Merged in https://github.com/handshake-org/hsd/commit/0da753980c0899667d52188883ccd728f9355e96